### PR TITLE
Vastly improved monopoly bot checking, and added a few missing bots

### PR DESCRIPTION
**Pull request by HcgRandon**
Hi, I'm the creator of Ayana, I wanted to help you out a little bit. I have improved your monopoly checking code quite a bit and added a few missing bots.

Enjoy 👍

### CODE REVIEW

**Comment by ducc** [comment]
**REMOVE DABBOT OR PAY THE PRICE!!!!!!! I WILL NOT TAKE THIS KIND OF ABUSE!!!!!!!!!**

**Comment by ducc** [comment]
**MY TEAM OF HIGHLY TRAINED HACKERS ARE NOW APPROACHING YOUR LOCATION-- GOOD LUCK, KIDDO**

**Comment by ducc** [comment]
**THE TEAM APPROACHES**

File: inhibitors/monopoly.js [hunk 1]
```diff
@@ -4,24 +4,29 @@ exports.conf = {
...
+ { id: '159985870458322944', name: 'Mee6' },
+ { id: '189702078958927872', name: 'Eris' },
+ { id: '153613756348366849', name: 'TypicalBot' },
+ { id: '185476724627210241', name: 'Ayana' },
+ { id: '195244363339530240', name: 'KawaiiBot' },
+ { id: '150300454708838401', name: 'Aethex' },
+ { id: '110462073074388992', name: 'Wild' },
+ { id: '116275390695079945', name: 'Nadeko' },
+ { id: '204255221017214977', name: 'YAGPDB' },
+ { id: '184405311681986560', name: 'Fredboat' },
```
Comment (napstr) [inline inhibitors/monopoly.js]: It's called Fred**B**oat. Please fix.

Comment (HcgRandon) [reply inhibitors/monopoly.js]: On it!

File: inhibitors/monopoly.js [hunk 2]
```diff
@@ -4,24 +4,29 @@ exports.conf = {
...
+ { id: '159985870458322944', name: 'Mee6' },
+ { id: '189702078958927872', name: 'Eris' },
+ { id: '153613756348366849', name: 'TypicalBot' },
+ { id: '185476724627210241', name: 'Ayana' },
+ { id: '195244363339530240', name: 'KawaiiBot' },
+ { id: '150300454708838401', name: 'Aethex' },
+ { id: '110462073074388992', name: 'Wild' },
+ { id: '116275390695079945', name: 'Nadeko' },
+ { id: '204255221017214977', name: 'YAGPDB' },
+ { id: '184405311681986560', name: 'FredBoat' },
```
Comment (Frederikam) [inline inhibitors/monopoly.js]: This is a typo. I think you meant `150376112944447488`

**Comment by aequasi** [comment]
Why do it this way? Just parse carbonitex or discordbots for all bots with > 10000 servers

**Comment by Oliyy** [comment]
We'd be happy to supply your API parsing needs at https://discordbots.org 👍

**Comment by HcgRandon** [comment]
@aequasi I thought about it but it might be 2 much for a first PR 🤔

File: inhibitors/monopoly.js (hunk 2 above)
Comment (HcgRandon) [reply inhibitors/monopoly.js]: I thought the music one was the larger one, maybe they should both be on the list, what do you think?

**Comment by DasWolke** [comment]
LGTM

File: inhibitors/monopoly.js [hunk 3]
```diff
@@ -4,24 +4,29 @@ exports.conf = {
...
+ { id: '270198738570444801', name: 'Dabbot' },
```
Comment (itslukej) [inline inhibitors/monopoly.js]: *dabBot

**Comment by HcgRandon** [comment]
@aequasi also this bit of code seems to run on every message / command to the bot. Might cause API spam

**Comment by aequasi** [comment]
Memoize it!

**Comment by Lovinity** [comment]
Thanks everyone but this repository has been deprecated. I apparently didn't delete it when I should have haha. The bot now resides at Lovinity/lovinibot . Though the code is public, the bot itself will not work in other guilds as it was customised specifically for LoveyCube and the LoveyCube website.

**Comment by aequasi** [comment]
Well that sucks.

File: inhibitors/monopoly.js (hunk 3 above)
Comment (napstr) [reply inhibitors/monopoly.js]: Actually, why hardcode the names at all, they should be just looked up in the guild, in case they have custom nicknames.

**Comment by HcgRandon** [comment]
*cries in Japanese*

**Comment by RobinJoh95** [comment]
Well that's a shame, all that work for nothing 😢

**Comment by aequasi** [comment]
*Cries in single spaces*

**Comment by itslukej** [comment]
I have contacted my lawyers regarding this situation.

**Comment by TheAkio** [comment]
*cries in german*

**Comment by Oliyy** [comment]
Would have made you a partnered bot on https://discordbots.org to be fair

**Pull request closed without merging**